pyproject.toml

```diff
@@ -104,4 +105,14 @@ fail-under = 5.6
 
 [tool.pylint.format]
 max-line-length = 88
-max-module-lines = 800
\ No newline at end of file
+max-module-lines = 800
+
+[tool.mypy]
+files = [
+    'noxfile.py',
+    'scripts/**/*.py',
+    # Incrementaly add ad fix files and paths until entire project is checked
```

Comment:
```suggestion
    # Incrementaly add fix files and paths until entire project is checked
```

Reply:
:+1: will integrate it in my change which addresses the failing python 3.8 build xD.